signal_scanner_bot/bin/run.py

```diff
@@ -26,6 +26,14 @@ def cli(debug: bool) -> None:
         for _log in ["requests", "oauthlib", "requests_oauthlib", "urllib3"]:
             logging.getLogger(_log).setLevel(logging.INFO)
 
+    log.info(
+        f"Loading Autoscan state: {'Enabled' if env.STATE.LISTENING else 'Disabled'}"
+    )
+    signal.send_message(
+        f"STARTUP\n{env.STATE.get_listening_status_notice()}",
+        env.LISTEN_GROUP,
+        group=True,
+    )
```

Comment:
I'm not the biggest fan of sending out a message every time the service reboots (because it can happen semi frequently sometimes as a result of the error in #23). I think the logging line should be sufficient!
```suggestion
```